examples/07-periodic_hills/periodic_hills.prm

```diff
@@ -79,7 +87,7 @@ end
 # FEM
 #---------------------------------------------------
 subsection FEM
-    set velocity order            = 2
```

Comment:
Is there a reason for this other than speed? It's ok if the example takes a while to run. I just wanted to keep the example using high-order mapping

Reply:
I know, as I said, this is not my final .prm file, but I still need to change it because of the new parameters :)